onnx/test/reference_evaluator_ml_test.py

```diff
@@ -829,22 +831,33 @@ def _get_test_tree_ensemble_regressor(
         return onx
 
     @unittest.skipIf(not ONNX_ML, reason="onnx not compiled with ai.onnx.ml")
-    def test_tree_ensemble_regressor(self):
+    @parameterized.expand(
+        [
+            (f"{agg}_{base_value}", base_value, agg)
+            for base_value in (None, [1.0])
+            for agg in ("SUM", "AVERAGE", "MIN", "MAX")
+        ]
+    )
+    def test_tree_ensemble_regressor(
+        self, name, base_value, agg
+    ):  # pylint: disable=unused-argument
```

Comment:
Use del to remove unused-arguments https://google.github.io/styleguide/pyguide.html#214-decision

Reply:
OK. Changed.